[cli] add encrypt and fetch-keys command (#281)
MystenLabs/seal: docs/UsingSeal.md

  Encapsulation: 841b3a59241e099e8b8d9cec1d531b1e8fe4b4170433e30d9aaa9fc764201f69e589a0b2a0e65bfb279d4b25ee1ce8141812bfb785abdb05134c3958f53c2e81e7bc06e5c1f1ebd7e489b5cf652216b13e6b7c2b13da70a4a7c05c3544a1ddf7
```

> Encrypt secret for a given Hex encoded unique identifier and seal policy package ID, using the specified Seal key server object IDs and network. The public keys are fetched using the public fullnode. 

Reviewer: ```suggestion
Encrypt a secret for a hex-encoded unique identifier and a Seal policy package ID. Use the specified key server object IDs and network. The CLI retrieves public keys from a public Full node.
```

```shell
$ cargo run --bin seal-cli encrypt --secret 045a27812dbe456392913223221306 \
    --id 0000 \
    -p 0xfaeabd7f317dd7ae40d83b73cfa68b92795f48540d03f1232b33207e22d0a62f \
    -t 2 \
    -k 0x73d05d62c18d9374e3ea529e8e0ed6161da1a141a94d3f76ae3fe4e99356db75,0xf5d14a81a982144ae441cd7d64b09027f116a468bd36e7eca494f750591623c8 \
    -n testnet

Encrypted object:
<ENCODED_ENCRYPTED_OBJECT>
```

> Fetch keys with the given encoded fetch key request, for a given threshold, key server object IDs and network. 

Reviewer: ```suggestion
Fetch keys for the encoded request. Provide the threshold, key server object IDs, and network.
```
